psi4/driver/driver_cbs.py

```diff
@@ -25,34 +25,152 @@
 #
 # @END LICENSE
 #
+"""Plan, run, and assemble QC tasks to obtain composite method, basis, & options treatments.
+
+===
+CBS
+===
+
+cbs_text_parser()
+-----------------
+
+    _parse_cbs_gufunc_string()
+    --------------------------
+    * break user string into paired method and basis stages
+
+* transform user string into cbs kwargs inc'l basic cbs_metadata
+
+
+----------------------------
+CompositeComputer.__init__()
+----------------------------
+
+    _process_cbs_kwargs()
+    ---------------------
+    * transform user kwargs into trial cbs_metadata format (aka dict spec)
```

Comment:
But these may not be user kwargs, if they were generated by `cbs_text_parser`?

Reply:
yes, user specified but not necessarily user kwargs, if processed through `cbs_text_parser`